config/default.yml

```diff
@@ -546,3 +546,8 @@ PreRebase:
     requires_files: false
     required: false
     quiet: false
+
+  MergedCommits:
+    enabled: false
+    description: 'Checking for commits that have already been merged'
+    dest_branches: ['master']
```

Comment:
I think we're fine to just call this `branches`, given the context and given no other name really jumps out.